dxr/plugins/clang/tests/test_callers.py

```diff
@@ -36,15 +36,15 @@ def test_no_caller(self):
         self.found_nothing('callers:orphan')
 
     def test_one_caller(self):
-        self.found_line_eq('callers:called_once', '<b>called_once</b>();')
+        self.found_line_eq('callers:called_once', '<b>called_once(5)</b>;')
```

Comment:
Let's add to the docstring here that we're also testing the highlighting of arguments. Do that, and r+!